Apps/Contoso.WinUI.Desktop.Demo/Contoso.WinUI.Desktop.Demo/app.manifest

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="Contoso.WinUI.Desktop.Demo.app"/>
+
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <!-- The combination of below two tags have the following effect:
+           1) Per-Monitor for >= Windows 10 Anniversary Update
+           2) System < Windows 10 Anniversary Update
```

Comment:
Not sure if it is clear. What is happening if `System < Windows 10 Anniversary Update`?

Reply:
jfyi
this is just a default comment that is added after the application is created